cog.yaml

```diff
@@ -2,12 +2,6 @@ build:
   gpu: true
   cuda: "11.6"
   python_version: "3.10"
-  python_packages:
-    - "diffusers==0.11.1"
-    - "torch==1.13.0"
-    - "ftfy==6.1.1"
-    - "scipy==1.9.3"
-    - "transformers==4.25.1"
-    - "accelerate==0.15.0"
+  python_requirements: "requirements.txt"
```

Comment:
I think `python_requirements` is a thing now? See https://github.com/replicate/cog/pull/714

Using it here so I can manually `pip install -r requirements.txt` in the Actions workflow.